API/graphql-api/api-server.js

```diff
@@ -0,0 +1,28 @@
+import { ApolloServer, gql } from "apollo-server-express";

```

Comment:
Voeg even een readme.md hiervoor toe, maakt het duidelijk waarom je precies deze file in deze folder hebt gezet en waar dit voor dient. Is wat overzichtelijker